_schemaV2.graphql

```diff
@@ -24,460 +24,6 @@ type AggregationCount {
   name: String!
 }
 
-type AnalyticsArtist {
```

Comment:
These changes are from running a dump schema without the right stitching ENV vars locally.